src/bundle/Resources/config/services/menu.yml

```diff
@@ -128,3 +128,8 @@ services:
         public: true
         tags:
             - { name: knp_menu.menu_builder, method: build, alias: ezplatform_admin_ui.menu.url_edit.sidebar_right }
+
+    EzSystems\EzPlatformAdminUi\Menu\ChangePasswordRightSidebarBuilder:
+        public: true
+        tags:
+            - { name: knp_menu.menu_builder, method: build, alias: ezplatform_admin_ui.menu.change_password.sidebar_right }
```

Comment:
`ezplatform_admin_ui.menu.change_password.sidebar_right` it should have `user` in here somewhere.